frontend/src/app/supportAdmin/DeviceType/DeviceForm.tsx

```diff
@@ -207,29 +323,39 @@ const DeviceForm = (props: Props) => {
                   />
                 </div>
               </div>
+              {device?.supportedDiseaseTestPerformed?.map((disease, index) =>
+                createSupportedDiseaseRow(index)
+              )}
               <div className="grid-row grid-gap">
                 <div
                   className="tablet:grid-col"
                   style={{ marginBottom: "56px" }}
                 >
-                  <MultiSelect
-                    key={device?.internalId}
-                    label="Supported diseases"
-                    name="supportedDiseases"
-                    onChange={(supportedDiseases) => {
-                      updateDeviceAttribute(
-                        "supportedDiseases",
-                        supportedDiseases
-                      );
+                  <Button
+                    className="margin-top-2"
+                    onClick={() => {
+                      if (device?.supportedDiseaseTestPerformed) {
+                        let newSupportedDisease = [
+                          ...device?.supportedDiseaseTestPerformed,
+                        ];
+                        newSupportedDisease.push({
+                          supportedDisease: "",
+                          testPerformedLoincCode: "",
+                        });
+                        updateDeviceAttribute(
+                          "supportedDiseaseTestPerformed",
+                          newSupportedDisease
+                        );
+                      } else {
+                        updateDeviceAttribute("supportedDiseaseTestPerformed", [
+                          { supportedDisease: "", testPerformedLoincCode: "" },
+                        ]);
+                      }
                     }}
-                    options={props.supportedDiseaseOptions}
-                    initialSelectedValues={
-                      device?.supportedDiseases.length
-                        ? device?.supportedDiseases
-                        : undefined
-                    }
+                    variant="unstyled"
+                    label={"Add another disease"}
+                    icon="plus"
                     disabled={!device}
-                    required
                   />
                 </div>
               </div>
```

Comment:
Given that we are moving away from the Pill component to add the disease the new supported disease section should validate that we don't set the same disease. This is how the pill component worked so it makes sense we account for this scenario in the new implementation as well.

Reply:
It might make sense to tack this onto [Elisa's form validation ticket](https://app.zenhub.com/workspaces/simplereport-growth--engagement-605b43a6ceb92c000f86279a/issues/gh/cdcgov/prime-simplereport/5106)